dxr/plugins/core.py

```diff
@@ -313,6 +320,13 @@ def needles_by_line(self):
             yield [('number', number),
                    ('content', text)]
 
+    def is_interesting(self):
+        """
+        Core is responsible for putting files in the index,
+        so everything is interesting
+        """
+        return True
```

Comment:
Since we don't want to show all binary files yet, only images and text files are interesting.